client/src/graphics/Stack.lua

```diff
@@ -996,4 +996,13 @@ function Stack:drawPanels(garbageImages, shockGarbageImages, shakeOffset)
 
   panelSet:drawBatch()
   prof.pop("Stack:drawPanels")
+end
+
+function Stack:initializeGraphics()
+  self.multi_prestopQuad = GraphicsUtil:newRecycledQuad(0, 0, self.theme.images.IMG_multibar_prestop_bar:getDimensions())
+  self.multi_stopQuad = GraphicsUtil:newRecycledQuad(0, 0, self.theme.images.IMG_multibar_stop_bar:getDimensions())
+  self.multi_shakeQuad = GraphicsUtil:newRecycledQuad(0, 0, self.theme.images.IMG_multibar_shake_bar:getDimensions())
+  self.multiBarFrameCount = self:calculateMultibarFrameCount()
+
+  characters[self.character]:initializeStackAnimation(self)
```

Comment:
Pulling this out is not strictly necessary but as I've mentioned before, I want to isolate the engine longterm so we can run store verification server side without love and moving graphics initialization to the client side helps a bit.
The stack getting its own clone of the battle animation was simply an opportunity to touch the code in this manner.